core/src/main/java/com/sequenceiq/cloudbreak/controller/v4/AuditEventV4Controller.java

```diff
@@ -30,6 +32,7 @@
     private CloudbreakRestRequestThreadLocalService threadLocalService;
 
     @Override
+    @CheckPermissionByAccount(action = AuthorizationResourceAction.GET_AUDIT_EVENTS)
```

Comment:
I am not familiar with the context of auditing, but I think these checks should be based on resource crn and not on account id. Please see: [https://github.com/hortonworks/cloudbreak/tree/master/authorization-common](https://github.com/hortonworks/cloudbreak/tree/master/authorization-common)